web/package.json

```diff
@@ -44,6 +44,7 @@
     "husky": "^4.3.8",
     "jest": "^26.6.3",
     "lint-staged": "^10.5.3",
+    "moment": "^2.29.1",
```

Comment:
@Vishalsag is moment in use, because not seen use in this PR.

Reply:
yes it is used to convert date